components/suggestion/switch_channel_provider.jsx

```diff
@@ -277,6 +278,16 @@ function makeChannelSearchFilter(channelPrefix) {
             }
         }
 
+        if (splitPrefixBySpace.length > 1) {
+            let match = true;
+            splitPrefixBySpace.forEach((searchPrefix) => {
+                if (!searchString.toLowerCase().includes(searchPrefix)) {
+                    match = false;
+                }
+            });
+            return match;
```

Comment:
Hopefully this is easier to read, feel free to ignore if you don't feel it is the case :)
```suggestion
            const lowerCaseSearch = searchString.toLowerCase();
            return splitPrefixBySpace.every((searchPrefix) => {
                return lowerCaseSearch.includes(searchPrefix);
            });
```

Reply:
That is nice, Thank you @Willyfrog